R/language.R

```diff
@@ -20,8 +20,13 @@
 #' q3 <- quote(x + 1)
 #' is_lang(q3)
 #' is_call(q3)
+#'
+#' # A literal is either a scalar atomic vector or a name:
```

Comment:
Or should it have another name?

The concept that I was trying to represent is that these objects are self-quoting. E.g. if you quote a vector, you get a call to `c()`. But if you quote a scalar atomic vector, you get the scalar literally.